package.json

```diff
@@ -35,8 +36,6 @@
     "@web3-react/injected-connector": "^6.0.7",
     "ethers": "^5.6.2",
     "framer-motion": "^6.2.8",
-    "@web3-react/walletconnect-connector": "^6.2.13",
```

Comment:
please add these wallet connectors back!

Reply:
should be there now